src/Google.Cloud.Functions.Framework/GcfEvents/GcfConverters.cs

```diff
@@ -68,9 +76,16 @@ internal static async ValueTask<CloudEvent> ConvertGcfEventToCloudEvent(HttpRequ
             {
                 throw new CloudEventConverter.ConversionException($"Unexpected event type for function: '{gcfType}'");
             }
-            if (gcfType.StartsWith("providers/cloud.firestore/eventTypes/", StringComparison.Ordinal))
+
+            // Firestore and Firebase database GCF requests include a "params" element that maps wilcards in the resource
+            // template to their value in the changed document.
+            if (gcfType.StartsWith("providers/cloud.firestore/eventTypes/", StringComparison.Ordinal) ||
+                gcfType.StartsWith("providers/google.firebase.database/", StringComparison.Ordinal))
```

Comment:
silly question: Why does the Firestore check includes the "eventTypes" but the Firebase check doesn't?

Reply:
No particular reason. Will take it out of Firestore - I can't imagine we'd get anything *other* than eventTypes after the second slash.